crates/libs/core/windows.natvis

```diff
@@ -27,10 +27,10 @@
   </Type>
 
   <Type Name="windows_core::strings::hstring::HSTRING">
-    <Intrinsic Name="header" Expression="*((windows_core::strings::hstring::Header**)this)" ReturnType="windows_core::strings::hstring::Header *" />
-    <Intrinsic Name="is_empty" Expression="header() == nullptr" />
+    <Intrinsic Name="header" Expression="(windows_core::strings::hstring::Header *)__0.tag" ReturnType="windows_core::strings::hstring::Header *" />
```

Comment:
The `ReturnType` can be omitted as it is implied by the `Expression`. I don't know why it was initially introduced. At least in this iteration, it is redundant and confusing.

Reply:
Done